endpoints/api/organization.py

```
@@ -1019,6 +1020,18 @@ def post(self, orgname):
 
+        # Check for active organization mirror configuration
+        if features.ORG_MIRROR:
+            mirror = model.org_mirror.get_org_mirror_config(org)
+            if mirror is not None:
+                logger.warning(
+                    "Blocking proxy cache creation for org '%s': organization mirroring is configured",
+                    orgname,
+                )
+                raise request_error(
+                    "Cannot enable proxy cache: organization mirroring is already configured"
+                )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)